loafwallet/AppDelegate.swift

```diff
@@ -97,6 +103,18 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
             FirebaseApp.configure(options: fileOpts)
         #endif
     }
+    
+    /// Check Locale
+    func updateCurrentUserLocale(localeId: String) {
+          
+        let suffix = String(localeId.suffix(3))
+        
+        if suffix == "_US" {
```

Comment:
@kcw-grunt  we can simplify the expression like this:
`UserDefaults.userIsInUSA = suffix == "_US"`

Reply:
Hey @mosadialiou  this is to allow people that are in the US but speak other languages .  So, someone that speaks French and has their phone set for french but are in say, New York . This region code would be "fr_US".
So this code truncates the `fr`.

Reply:
@kcw-grunt yeah I understand. I am talking about the code. You can simplify it like I said!